docs/troubleshooting.md

```diff
@@ -116,3 +117,9 @@ In case a new file is added to the empty directory remotely, outside of the moun
 ### fuse: *fuseops.FallocateOp error: function not implemented or similar function not implemented errors
 
 This is an expected error for file operations unsupported in FUSE file system (details [here](https://github.com/GoogleCloudPlatform/gcsfuse/discussions/2386#discussioncomment-10417635)).
+
+### transport endpoint is not connected
+
```

Comment:
It might be good to what the error is and the reason before adding the solution.

Reply:
cl/642277208 seems to have most of the steps. Could we add similar here. We could also add above the less resources as an additional point.

Reply:
+1. this will be good for consistency also.

Reply:
Added in the form of cl/642277208